public/static/docs/command-reference/metrics/diff.md

```diff
@@ -0,0 +1,123 @@
+# metrics diff
+
+Show a table of changes in [metrics](/doc/command-reference/metrics#description)
+between two versions of the <abbr>DVC repository</abbr>.
+
+> This command requires that the <abbr>project</abbr> is a
+> [Git](https://git-scm.com/) repository.
+
+## Synopsis
+
+```usage
+usage: dvc metrics diff [-h] [-q | -v]
+                        [--targets [TARGETS [TARGETS ...]]]
+                        [-t TYPE] [-x XPATH] [-R] [--show-json]
+                        [a_ref] [b_ref]
+
+positional arguments:
+  a_ref     Git reference to the older version to compare
+            (defaults to HEAD)
+  b_ref     Git reference to the newer version to compare
+            (defaults to the current workspace including changes)
+```
+
+## Description
+
+The changes shown by this command includes the new value, and numeric difference
+(delta) from the previous value of metrics (with 3-digit accuracy). They're
+calculated between two commit SHA hashes, branch or tag names, etc.
+([Git references](https://git-scm.com/docs/revisions)) for all metrics in the
```

Comment:
revision?

Reply:
Yas, will update as part of https://github.com/iterative/dvc.org/pull/933#pullrequestreview-356892140 above.